parodos-model-api/src/test/java/com/redhat/parodos/workflow/task/shell/ShellRunnerTaskTest.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.parodos.workflow.task.shell;
+
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class ShellRunnerTaskTest {
+
+	ShellRunnerTask underTest;
+
+	WorkContext ctx;
+
+	@Before
+	public void setUp() {
+		underTest = new ShellRunnerTask();
+		ctx = new WorkContext();
+	}
+
+	@Test
+	public void executeWithOutput() {
+		UUID randomUUID = UUID.randomUUID();
+		ctx.put("command", "echo");
+		ctx.put("args", randomUUID);
+		var output = new StringBuilder();
+		underTest.outputConsumer = output::append;
+
+		assertThat(underTest.execute(ctx).getStatus()).isEqualTo(WorkStatus.COMPLETED);
+		assertThat(output.toString()).isEqualTo(randomUUID.toString());
```

Comment:
could we assert that the temp folder is deleted?

Reply:
following the comment above, we can count on the jvm to execute that. but for extra cation we can parametrize the call to pass in a temp dir ... wonder if its worth it.
Another though I had - writing  stuff into /tmp costs memory while /var/tmp is disk space only and I tend to think those executions could involve a lot of stuff written into the scratch space of the task .